packages/liquidator/src/liquidator.js

```diff
@@ -148,6 +148,210 @@ class Liquidator {
       at: "Liquidator",
       message: "Checking for liquidatable positions and preforming liquidations"
     });
+    // If an override is provided, use that price. Else, get the latest price from the price feed.
```

Comment:
This is the beginning of the new liquidatePositions function, which basically copied the old code but strips out 1inch